### DISCUSSION TITLE
Contributors: Extension Methods

### DISCUSSION THREAD

**Discussion by paulushub** [vvvv/SVG#1126] [Contributors]
### Description
> **Confession**: I am not a big fan of having a single public class in its own namespace, so this possibly may affect my views on this discussion.

Now, currently 
- There is a root directory file `Svg/Extensions,cs`, which defines a number of extension classes.
- There is also a folder `Svg/ExtensionMethods`, which contains an extension class (with public access), `UriExtensions.cs`.

### Question
Why is there a need to have this folder `Svg/ExtensionMethods`? 
**NOTE** Please there is no plan to remove this, as it  may result in breaking change (just a wish to understand the thought process here)
- It defines a separate namespace `Svg.ExtensionMethods`
- the class `UriExtensions` is public,.

**Comment by mrbean-bremen** [comment]
You may want to use discussions instead of issues for this kind of questions (maybe also for other discussion related stuff).

**Comment by paulushub** [comment]
@mrbean-bremen Got it, just trying to draw a quick attention 😄
